### DISCUSSION TITLE
How to use JSON_EXTRACT

### DISCUSSION THREAD

**Discussion by lbb00** [prisma/prisma#16238] [Q&A] [answered]
For example, in Sequelize I can use 

```javascript
where = {
  [Op.and]: literal(`JSON_EXTRACT(content, '$[*].form[*].value') like '%"${formValue}"%'`),
}
```
So is there a way to do that by Prisma?

**Comment by pantharshit00** [comment, marked as answer]
Hello @lbb00 

Calling raw functions in the client is not supported yet, you can follow https://github.com/prisma/prisma/issues/12279. You will need to use a raw query here if you want to use that SQL functions. 

But I think you can the client API iteself here to traverse the JSON documents using the `path` property in the JSON fields in the client API. Read more here: https://www.prisma.io/docs/concepts/components/prisma-client/working-with-fields/working-with-json-fields#filter-on-object-property

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
